connector/protobuf/src/test/scala/org/apache/spark/sql/protobuf/ProtobufFunctionsSuite.scala

```diff
@@ -2101,7 +2102,9 @@ class ProtobufFunctionsSuite extends QueryTest with SharedSparkSession with Prot
         queryContext = Array(ExpectedContext(
           fragment = s"to_protobuf(complex_struct, 42, '$testFileDescFile', map())",
           start = 10,
-          stop = 153))
+          // The subsequent test scenarios also follow the following rules to define `stop`:
+          // stop = (start - 1) + prefixLength + fileNameLength + suffixLength
+          stop = 9 + 33 + fileNameLength + 9))
```

Comment:
@itholic If you haven't started investigating yet, could you please help check if this fix is feasible? Thanks
also cc @HyukjinKwon